llvm/test/CodeGen/AIE/aie2/schedule/postpipeliner/conv2d_bf16-1.mir

```diff
@@ -12,8 +12,8 @@
 # derived from conv2d_bf16_0
 
 --- |
-  define dso_local void @gemm(ptr addrspace(5) noalias nocapture writeonly %d, ptr addrspace(6) noalias nocapture readonly %s, i32 noundef %n) local_unnamed_addr #0 {
-  ; CHECK-LABEL: gemm:
+  define dso_local void @conv2d(ptr addrspace(5) noalias nocapture writeonly %d, ptr addrspace(6) noalias nocapture readonly %s, i32 noundef %n) local_unnamed_addr #0 {
+  ; CHECK-LABEL: conv2d:
```

Comment:
I assume you'll squash that with the previous commit?

Reply:
Yes. that slipped through. I noticed another name change to be done.